packages/manager/modules/pci/src/projects/project/storages/databases/database/general-information/general-information.routing.js

```diff
@@ -177,6 +220,7 @@ export default /* @ngInject */ ($stateProvider) => {
         database.nodes.forEach((node) =>
           DatabaseService.stopPollingNodeStatus(node.id),
         ),
+      nodesPerRow: () => NODES_PER_ROW,
```

Comment:
Why not just import the constant where its needed instead of injecting it ? Is it planned to be exposed through API ?

Reply:
It is not planned to be exposed throught API as the number of nodes to show per row is a frontend rule only.
We need the constant in the template, should we bind it in the controller rather than injecting it in the routing ?